packages/assets-controllers/src/TokenDetectionController.ts

```diff
@@ -1,13 +1,17 @@
 import type { BaseConfig, BaseState } from '@metamask/base-controller';
-import { BaseController } from '@metamask/base-controller';
 import {
   safelyExecute,
   toChecksumHexAddress,
 } from '@metamask/controller-utils';
-import type { NetworkState } from '@metamask/network-controller';
+import type {
+  NetworkClientId,
+  NetworkController,
+  NetworkState,
+} from '@metamask/network-controller';
 import type { PreferencesState } from '@metamask/preferences-controller';
 import type { Hex } from '@metamask/utils';
 
+import { PollingControllerV1 } from '../../polling-controller/src/PollingController';
```

Comment:
`PollingControllerV1` import should pull from the `@metamask/polling-controller` package instead of a relative path?

```suggestion
} from '@metamask/network-controller';
import { PollingControllerV1 } from '@metamask/polling-controller';
import type { PreferencesState } from '@metamask/preferences-controller';
import type { Hex } from '@metamask/utils';

```